packs/vsphere/actions/vmwarelib/actions.py

```diff
@@ -20,6 +20,8 @@
 from pyVmomi import vim
 from st2actions.runners.pythonrunner import Action
 
+connectionitems = ['host', 'port', 'user', 'passwd']
```

Comment:
Minor style thing (not a blocker) for module level constants we usually use upper case, underscore separated variable names - e.g. `CONNECTION_ITEMS = ...`